src/app/features/ebic/index.js

```
@@ -0,0 +1,270 @@
+      .addCase(getEbicCategories.fulfilled, (state, action) => {
+        state.isFetching = false;
+        state.categories = action?.payload;
+      })
+      .addCase(getEbicCategories.rejected, (state, action) => {
+        state.isFetching = false;
+        state.error = action?.payload;
+      })
+      .addCase(getEbicByCategory.pending, (state) => {
+        state.isEbicFetching = true;
+      })
+      .addCase(getEbicByCategory.fulfilled, (state, action) => {
```

ESLint

[error] 'action' is defined but never used.

(no-unused-vars)